src/pages/settings/PasswordPage.js

```diff
@@ -143,14 +137,17 @@ class PasswordPage extends Component {
                                 textContentType="password"
                                 value={this.state.newPassword}
                                 onChangeText={newPassword => this.setState({newPassword})}
-                                onFocus={() => this.setState({isPasswordRequirementsVisible: true})}
                                 onBlur={() => this.onBlurNewPassword()}
                             />
-                            {this.state.isPasswordRequirementsVisible && (
-                                <Text style={[styles.textLabelSupporting, styles.mt1]}>
-                                    {this.props.translate('passwordPage.newPasswordPrompt')}
-                                </Text>
-                            )}
+
+                            <Text style={[
+                                styles.textLabelSupporting, styles.mt1,
```

Comment:
Can you put each of these on their own line?

Reply:
Can you move the style prop to its own line and indent accordingly? There's an example in the `// Good` section of https://github.com/Expensify/App/blob/main/STYLING.md#when-to-create-a-new-style

Reply:
Okay. Thanks for the share. I wasn't sure if every style will be moved to a new line. Do you think this should be added as a lint config?

Reply:
That sounds like a good improvement, I think it's worth bringing up in [slack](https://expensify.slack.com/archives/C01GTK53T8Q/p1631287040167100) to at least get a discussion going, but we'll keep that separate from this PR.